Description: Fix build for Debian GNU/kFreeBSD
 Don't use clang by default, and use GNU standard library linking flags
Author: Ximin Luo <infinity0@pwned.gg>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/config/nim.cfg
+++ b/config/nim.cfg
@@ -73,13 +73,11 @@
 @end
 
 @if unix:
-  @if not bsd:
-    # -fopenmp
-    gcc.options.linker = "-ldl"
-    gcc.cpp.options.linker = "-ldl"
-    clang.options.linker = "-ldl"
-    tcc.options.linker = "-ldl"
-  @end
+  # -fopenmp
+  gcc.options.linker = "-ldl"
+  gcc.cpp.options.linker = "-ldl"
+  clang.options.linker = "-ldl"
+  tcc.options.linker = "-ldl"
   @if bsd or haiku:
     # BSD got posix_spawn only recently, so we deactivate it for osproc:
     define:useFork
@@ -102,7 +100,7 @@
   @end
 @end
 
-@if macosx or freebsd:
+@if macosx:
   cc = clang
   tlsEmulation:on
   gcc.options.always = "-w"
